### ISSUE TITLE
Unicode boolean variables not handled?

### ISSUE THREAD

**Issue by pfuntner** [hgomersall/python-boolean#1]
I ran into an issue where I was supplying boolean variables in a list of unicode strings but they weren't being processed up properly by `parse_boolean_function_string()`.

```
In [13]: func = parse_boolean_function_string('blah', [u'blah'])

In [14]: func
Out[14]: u'blah'

In [15]: func = parse_boolean_function_string('blah', ['blah'])

In [16]: func
Out[16]: <function psirt_utils.python_boolean.logic.<lambda>>

In [17]:
```

It's easy enough for me to turn the unicode strings into regular strings but is this working as designed?  I wasted a bit of debugging trying to figure out what's wrong.  I'm used to Python and packages not caring much about the difference between a string and a unicode string.  Python thinks strings and unicode strings with the same data are equal:

```
In [17]: u'blah' == 'blah'
Out[17]: True
```

I suspect python-boolean is looking at the types and doesn't treat string and unicode the same.

**Comment by hgomersall** [comment]
Your suspicion is correct. If you have a fix, it would be gratefully merged! :)

**Comment by hgomersall** [comment]
See line 206 of logic.py...

**Comment by pfuntner** [comment]
Fixed with #2
